src/components/seekbar/seekbar.js

```diff
@@ -99,28 +97,6 @@ class SeekBarControl extends Component {
     }
   }
 
-  /**
-   * onTap event handler
-   *
-   * @param {Event} e - onClick event
-   * @returns {void}
-   * @memberof SeekBarControl
-   */
-  onTap(e: Event): void {
-    if (!this.props.isMobile) {
-      return;
-    }
-    const oldTime = this.props.player.currentTime;
-    const newTime = this.getTime(e);
```

Comment:
won;t this kill the tap action? we support both dragging and tapping the seekbar. Did you check?

Reply:
Yes

Reply:
Yes on which one of the questions? :-)

Reply:
Lol, of course I checked. It's support dragging, the problem was that on click and ontouchend did same thing..